spec/controllers/challenge_assignments_controller_spec.rb

```diff
@@ -0,0 +1,29 @@
+require 'spec_helper'
+
+describe ChallengeAssignmentsController do
+  include LoginMacros
+  include RedirectExpectationHelper
+  
+  before(:each) do
```

Comment:
Right -- that's because it was like this:
```
      it 'error to login page' do
         let(:collection) { create(:collection) }
         get :no_challenge, collection_id: collection.name
          it_redirects_to_with_error(new_user_session_path, "Sorry, you don't have permission to access the page you were trying to reach. Please log in.")
        end
```

It can't be in the `it`, either. For this, the `let` should be directly after `include RedirectExpectationHelper`.